MANIFEST.in

```diff
@@ -1,3 +1,4 @@
 include CHANGES.md
 include COPYING
 include README.md
+recursive-include pyfakefs/tests/fixtures *
```

Comment:
Thanks - I missed that after adding that excel file! This also applies to the binary distribution, which also contains the tests, and will not be fixed by this change... Though I wonder if it actually makes sense to have the tests in the wheel.   
Anyway, if not changing this, the fixtures need also be added to the wheel package, if you don't mind.  
Also, can you please add an entry in the [release notes](https://github.com/pytest-dev/pyfakefs/blob/main/CHANGES.md) (under a new heading "Unreleased", as there have been no changes after todays's release yet)?

Reply:
Right, sorry, it didn't occur to me that tests are installed too. I'll look at how to do it.

Reply:
Done and done. To be more precise, I've done that via enabling `include_package_data` so that all data files from package directories are installed — lemme know if you prefer another approach.